class.jetpack-cli.php

```diff
@@ -304,8 +305,43 @@ public function reset( $args, $assoc_args ) {
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 				break;
 			case 'prompt':
-				WP_CLI::error( __( 'Please specify if you would like to reset your options, or modules', 'jetpack' ) );
+				WP_CLI::error( __( 'Please specify if you would like to reset your options, modules or sync-checksum', 'jetpack' ) );
 				break;
+			case 'sync-checksum':
+				global $wpdb;
+				$option = 'jetpack_callables_sync_checksum';
+				if ( is_multisite() && function_exists( 'get_sites' ) ) {
+					$sites = get_sites( array( 'number' => 1000 ) );
+					$count_fixes = 0;
+					foreach ( $sites as $site ) {
+						switch_to_blog( $site->blog_id );
+						$count = (int) $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->options WHERE option_name = %s", $option ) );
+						if ( $count > 1 ) {
+							delete_option( $option );
+							WP_CLI::line( sprintf( __( 'Deleted %s %s options from %s', 'jetpack' ), $count, $option, "{$site->domain}{$site->path}" ) );
```

Comment:
I wonder if we could make the messages more relevant to the fact that we're resetting the sync checksum. Seeing that message might make me think I have run the wrong query (a `wp option delete` command comes to mind 😱 ).

Reply:
I am hoping that the message would read 
`Deleted 100 jetpack_callables_sync_checksum options from example.com`. Which should be clear in terms of what happend. Haveing said that I would be happy to change to make this more clear.